src/openforms/submissions/models/submission.py

```diff
@@ -241,6 +242,15 @@ class Submission(models.Model):
         "submissions.Submission", on_delete=models.SET_NULL, null=True, blank=True
     )
 
+    language_code = models.CharField(
+        _("language code"),
+        max_length=2,
+        default=settings.LANGUAGE_CODE,
+        help_text=_(
+            "The code (ISO 639-1 format) of the language used to fill in the Form."
```

Comment:
As remarks in Chris' PR, this is not an ISO-639-1 code. he found out it's RFC5646 :)